### PR TITLE
Fixes #792 - Add CCDS-R for NODE Based Displays

### LINKED ISSUE #792: Incorporate CCDS-R for NODE Based Displays
## What is the current situation?
EuroScope C-Mode is sometimes wrong, doesn't reflect the real system completely.

## What is the proposed change?
Incorporate CCDS-R plugin for all NODE based displays, CCDS-R plugin by @Quassbutreally

## Reference materials (e.g. screenshots)
https://github.com/quassbutreally/EuroScope-CCDS-R

### PR DESCRIPTION
**Pull request by SamLefevre**
Fixes #792

# Summary of changes

Add CCDS-R plugin and modify tags

### CODE REVIEW

File: UK/Data/Plugin/CCDS-R/Settings/conversions.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    "callsigns": {
+		"0003": "HLE60",
+		"0014": "HLE21",
+		"0015": "HLE07",
+		"0016": "HLE24",
+		"0017": "HLE27",
+		"0031": "HLE55",
+		"0027": "FIRJOIN",
+        "1177": "FIS",
```
Comment (luke11brown) [inline UK/Data/Plugin/CCDS-R/Settings/conversions.json:17]: As we'll be covering Scottish, I'd suggest putting the Scottish FIS code here as well (7401)

File: UK/Area/London TC/TC Bandbox.prf [hunk 2]
```diff
@@ -47,6 +47,7 @@ Plugins	Plugin3	\..\..\Data\Plugin\VFPC.dll
 Plugins	Plugin4	\..\..\Data\Plugin\VCH.dll
 Plugins	Plugin5	\..\..\Data\Plugin\TopSky_NODE\TopSky.dll
 Plugins	Plugin5Display0	Standard ES radar screen
+Plugins	Plugin6	\..\..\Data\Plugin\CCDS-R\CCDS-R.dll
```
Comment (luke11brown) [inline UK/Area/London TC/TC Bandbox.prf:50]: Missing Plugindisplay lines, or does it need to be allowed to draw?

File: UK/Area/London TC/TC East.prf [hunk 3]
```diff
@@ -42,12 +42,17 @@ Plugins	Plugin0	\..\..\Data\Plugin\vSMR\vSMR.dll
 Plugins	Plugin0Display0	SMR radar display
 Plugins	Plugin1	\..\..\Data\Plugin\UKControllerPlugin.dll
 Plugins	Plugin1Display0	Standard ES radar screen
+Plugins	Plugin0	\..\..\Data\Plugin\vSMR\vSMR.dll
+Plugins	Plugin0Display0	SMR radar display
+Plugins	Plugin1	\..\..\Data\Plugin\UKControllerPlugin.dll
+Plugins	Plugin1Display0	Standard ES radar screen
```
Comment (luke11brown) [inline UK/Area/London TC/TC East.prf:48]: Duplicated stuff

File: UK/Area/London TC/TC Bandbox.prf (hunk 2 above)
Comment (SamLefevre) [reply UK/Area/London TC/TC Bandbox.prf:50]: Nah only tag items, doesn't need to draw on the display

File: UK/Data/Settings/Tags.txt [hunk 4]
```diff
@@ -415,13 +415,13 @@ TAGITEM:22::1:0:6:39:TopSky plugin:0:1:TopSky plugin:TopSky plugin:8
...
 TAGITEM:0::0:0:0:0::0:1:::1
-TAGITEM:3::0:0:136:0::0:1:TopSky plugin::1
+TAGITEM:3::0:0:0:0::0:1:::1
 TAGITEM:4::0:1:12:12::0:1:TopSky plugin:TopSky plugin:1
 TAGITEM:52::0:1:26:59::0:1::TopSky plugin:5
 TAGITEM:0::0:0:0:0::0:1:::1
 TAGITEM:101::0:1:0:0:UK Controller Plugin:0:1:::1
 TAGITEM:1:G:0:1:0:0::4:1:::1
-TAGITEM:40::0:0:135:0::0:1:TopSky plugin::1
+TAGITEM:40::0:0:0:0::0:1:::1
```
Comment (luke11brown) [inline UK/Data/Settings/Tags.txt:424]: NODE tag family starts at line 464. This will amend other tag families.

**Comment by SamLefevre** [comment]
if that doesnt fix it then no idea.

**Review by luke11brown — APPROVED** [review]

**Pull request merged**